src/components/application_manager/src/commands/hmi/on_exit_application_notification.cc

```diff
@@ -37,6 +37,7 @@
 #include "application_manager/message_helper.h"
 #include "interfaces/MOBILE_API.h"
 #include "interfaces/HMI_API.h"
+#include "functional_module/plugin_manager.h"
```

Comment:
@dev-gh probably this include also should be wrapped with `ifdef's` because they will be redundant if `SDL_REMOTE_CONTROL` will be OFF

Reply:
@dev-gh not agree